### PR TITLE
Adding Microsoft.EdgeZones to azure-sdk-for-net for public preview release version 2024-04-01-preview

### PR DESCRIPTION
**Pull request by jasonurban1**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by jasonurban1** [comment]
I have not written tests yet.  We were hoping we could get in for next months SDK release so due date Friday.  Can this be merged before March 12th with testcases not yet written?

File: sdk/edgezones/Azure.ResourceManager.EdgeZones/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure EdgeZones management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/edgezones/Azure.ResourceManager.EdgeZones/README.md:3]: Describe your service here

File: sdk/edgezones/Azure.ResourceManager.EdgeZones/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/edgezones/Azure.ResourceManager.EdgeZones/CHANGELOG.md:3]: Set expected release date by replacing the `Unreleased`, the format is `yyyy-mm-dd`.

File: sdk/edgezones/Azure.ResourceManager.EdgeZones/README.md (hunk 1 above)
Comment (jasonurban1) [reply sdk/edgezones/Azure.ResourceManager.EdgeZones/README.md]: Added description.

File: sdk/edgezones/Azure.ResourceManager.EdgeZones/CHANGELOG.md (hunk 2 above)
Comment (jasonurban1) [reply sdk/edgezones/Azure.ResourceManager.EdgeZones/CHANGELOG.md]: Exact release data set to 2024-04-26.

**Comment by jasonurban1** [comment]
Between Saturday and Today changes have been made that break the dotnet build /t:GenerateCode generated code.  Instead of continuing with this branch I created a new off of main and reran steps https://eng.ms/docs/products/azure-developer-experience/develop/sdk-generate-mgmt-dotnet

**Pull request closed without merging**